.github/workflows/build_native_binaries.yml

```diff
@@ -0,0 +1,80 @@
+on: workflow_dispatch
```

Comment:
Note: Only building what's necessary for CI right now, will add more targets soon